spec/models/protocol/valid?_spec.rb

```diff
@@ -38,6 +38,8 @@
     end
   end
 
-  it { is_expected.to validate_numericality_of(:indirect_cost_rate).is_greater_than_or_equal_to(1) }
-  it { is_expected.to validate_numericality_of(:indirect_cost_rate).is_less_than_or_equal_to(1000) }
+  if Setting.find_by_key('use_indirect_cost').value
```

Comment:
After discussing with @Stuart-Johnson, we should not use an `if` condition here, but instead a `before :each do` block where `use_indirect_cost` is turned on.

```
before :each do
  stub_config("use_indirect_cost", true)
end
```